mod-data-import-converter-storage-server/src/main/java/org/folio/rest/impl/DataImportProfilesImpl.java

```diff
@@ -230,7 +228,7 @@ public void deleteDataImportProfilesJobProfilesById(String id, String lang, Map<
                                                       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     vertxContext.runOnContext(v -> {
       try {
-        if (canDeleteOrUpdateProfile(id, JOB_PROFILES)) {
```

Comment:
Why did you change the delete endpoint? The issue is about updating...

Reply:
Yes, I removed authority job profile id from JOB_PROFILES array. We are able to update it, but not delete. So that, I had to  refactor the logic in delete API